src/views/v2/month/calendar-body/day.php

```diff
@@ -88,15 +102,47 @@ class="tribe-events-calendar-month__day-date-daynum"
 		<?php if ( ! empty( $day['featured_events'] ) ): ?>
 			<em
 				class="tribe-events-calendar-month__mobile-events-icon tribe-events-calendar-month__mobile-events-icon--featured"
-				aria-label="<?php esc_attr_e( 'Has featured events', 'the-events-calendar' ); ?>"
-				title="<?php esc_attr_e( 'Has featured events', 'the-events-calendar' ); ?>"
+				aria-label="<?php
+					echo esc_attr(
+						sprintf(
+							/* translators: %s: Events (plural). */
+							__( 'Has featured %s', 'the-events-calendar' ),
+							$events_label_plural
+						)
+					);
+				?>"
+				title="<?php
+					echo esc_attr(
+						sprintf(
+							/* translators: %s: Events (plural). */
+							__( 'Has featured %s', 'the-events-calendar' ),
+							$events_label_plural
+						)
+					);
+				?>"
```

Comment:
* :no_entry_sign: Line indented incorrectly; expected 6 tabs, found 4
* :no_entry_sign: Closing PHP tag must be on a line by itself

<sub>hash: e50381091ffd5b6f361b547e1e74a119</sub>